tests/gordo_components/builder/test_mlflow_utils.py

```diff
@@ -14,143 +15,213 @@
+                    ),
+                    "fold-1-train-end": Timestamp(
+                        "2019-01-15 22:30:00+0000", tz="UTC", freq="10T"
+                    ),
+                    "fold-1-n-train": 2118,
+                    "fold-1-test-start": Timestamp(
+                        "2019-01-15 22:40:00+0000", tz="UTC", freq="10T"
+                    ),
+                    "fold-1-test-end": Timestamp(
+                        "2019-01-30 15:00:00+0000", tz="UTC", freq="10T"
+                    ),
+                    "fold-1-n-test": 2115,
+                    "fold-2-train-start": Timestamp(
+                        "2019-01-01 05:40:00+0000", tz="UTC", freq="10T"
+                    ),
+                    "fold-2-train-end": Timestamp(
+                        "2019-01-30 15:00:00+0000", tz="UTC", freq="10T"
+                    ),
+                    "fold-2-n-train": 4233,
+                    "fold-2-test-start": Timestamp(
+                        "2019-01-30 15:10:00+0000", tz="UTC", freq="10T"
+                    ),
+                    "fold-2-test-end": Timestamp(
+                        "2019-02-14 07:30:00+0000", tz="UTC", freq="10T"
+                    ),
+                    "fold-2-n-test": 2115,
+                    "fold-3-train-start": Timestamp(
+                        "2019-01-01 05:40:00+0000", tz="UTC", freq="10T"
+                    ),
+                    "fold-3-train-end": Timestamp(
+                        "2019-02-14 07:30:00+0000", tz="UTC", freq="10T"
+                    ),
+                    "fold-3-n-train": 6348,
+                    "fold-3-test-start": Timestamp(
+                        "2019-02-14 07:40:00+0000", tz="UTC", freq="10T"
+                    ),
+                    "fold-3-test-end": Timestamp(
+                        "2019-03-01 00:00:00+0000", tz="UTC", freq="10T"
+                    ),
+                    "fold-3-n-test": 2115,
                 },
             },
+            "feature-thresholds": [0.5609058195202434, 0.5959327804053265],
+            "aggregate-threshold": 1.2813764724006529,
+            "scaler": "RobustScaler(copy=True, quantile_range=(25.0, 75.0), with_centering=True,\n             with_scaling=True)",
+            "base_estimator": "Pipeline(memory=None,\n         steps=[('step_0',\n                 PCA(copy=True, iterated_power='auto', n_components=None,\n                     random_state=None, svd_solver='auto', tol=0.0,\n                     whiten=False)),\n                ('step_1',\n                 MultiOutputRegressor(estimator=LinearRegression(copy_X=True,\n                                                                 fit_intercept=True,\n                                                                 n_jobs=None,\n                                                                 normalize=False),\n                                      n_jobs=None))],\n         verbose=False)",
```

Comment:
Is all this _really_ necessary? Not to say it should be fixed in this PR, but the fact that so much updating is required for a 'small' change like simply inserting start/end dates per fold requires all this adjustment for ML flow stuff to work seems excessive... :fearful:

Reply:
I made an issue for it here #791

Reply:
Cool, I think that'll work nicely when we sort that out.. maybe I'll be less afraid of doing stuff in this module. :wink: